frontend/src/metabase/visualizations/components/TableInteractive/TableInteractive.jsx

```diff
@@ -903,7 +903,7 @@ class TableInteractive extends Component {
           }
         >
           <QueryColumnInfoPopover
-            placement="bottom-start"
+            position="bottom-start"
```

Comment:
`QueryColumnInfoPopover` accepts `position`, not `placelement`

https://github.com/metabase/metabase/blob/master/frontend/src/metabase/components/MetadataInfo/ColumnInfoPopover/ColumnInfoPopover.tsx#L10

Reply:
Oh my...